packages/core-components/src/layout/Header/Header.tsx

```diff
@@ -213,7 +216,14 @@ export const Header = ({
           />
           <SubtitleFragment classes={classes} subtitle={subtitle} />
         </div>
-        <div className={classes.rightItemsBox}>{children}</div>
+        <Grid
+          component="div"
+          container
+          className={classes.rightItemsBox}
```

Comment:
Can we remove some styles from `rightItemsBox` after switching to `Grid`? Since it uses flexbox by default, some of these might no longer be necessary. I wonder if the `div className={classes.leftItemBox}` should be a `Grid` component too 🤔